### ISSUE TITLE
How to use setOperator correctly?

### ISSUE THREAD

**Issue by Happypig375** [jiggzson/nerdamer#258]
```js
nerdamer.setOperator('°', 'degree', 6, true, false, true, function(s){
    var _ = nerdamer.getCore().PARSER;
    return _.divide(_.multiply(s.clone(), Math.PI), 180)
});
nerdamer('90°') //Object has no method 'operation'
```
What did I do wrong?

**Comment by jiggzson** [comment]
You didn't do anything wrong. I forgot to pass along your function to the parser. Let's hold off on this until I can fix this since it's not quite ready. My access to a computer is somewhat limited right now but I'll tackle this one once I have full access to a computer.

**Comment by Happypig375** [comment]
> Let's hold off on this until I can fix this since it's not quite ready. 

Fixed in 0.7.13. Yay!
